app/src/main/java/com/dd3boh/outertune/MainActivity.kt

```diff
@@ -1157,6 +1157,8 @@ class MainActivity : ComponentActivity() {
                 }
             }
         }
+
+        downloadUtil.resumeDownloadsOnStart(this@MainActivity)
```

Comment:
This is called on every screen rotation since the main activity re-composes itself(?). This might be avoided in a launched effect??? I'm not too sure how to solve this one